src/modules/imports/import.rs

```diff
@@ -145,21 +145,31 @@ impl SyntaxModule<ParserMetadata> for Import {
             Err(_) => {
                 token(meta, "{")?;
                 let mut exports = vec![];
-                loop {
-                    let tok = meta.get_current_token();
-                    let name = variable(meta, variable_name_extensions())?;
-                    let alias = match token(meta, "as") {
-                        Ok(_) => Some(variable(meta, variable_name_extensions())?),
-                        Err(_) => None
-                    };
-                    exports.push((name, alias, tok));
-                    match token(meta, ",") {
-                        Ok(_) => {},
-                        Err(_) => break
+                if token(meta, "}").is_err() {
+                    loop {
+                        let tok = meta.get_current_token();
+                        let name = variable(meta, variable_name_extensions())?;
+                        let alias = match token(meta, "as") {
+                            Ok(_) => Some(variable(meta, variable_name_extensions())?),
+                            Err(_) => None
+                        };
+                        exports.push((name, alias, tok));
+                        if token(meta, "}").is_ok() {
+                            break;
+                        }
+                        match token(meta, ",") {
+                            Ok(_) => {
+                                if token(meta, "}").is_ok() {
+                                    break
+                                }
+                            }
+                            Err(_) => {
+                                return error!(meta, meta.get_current_token(), "Expected ',' or '}' after import");
+                            }
+                        }
                     }
                 }
```

Comment:
Please insert a warning here for developers to quickly spot an empty import statement:
```
import { } from "std/text"
```
With a warning message like:
```suggestion
                } else {
                    let message = Message::new_warn_at_token(meta, Some(self.token_import.clone()))
                        .message("Empty import statement")
                    meta.add_message(message);
                }
```

Reply:
Committed with some modification: i) Adding semicolon, and ii) Removing `Some()`. (E0308)

And I have a question. Is there a way to make a unit test for warning?

Reply:
Uhhh not for now but that should be implemented at some point. I'll make an issue about it